politeiawww/cmd/politeiawwwcli/commands/edituser.go

```diff
@@ -32,17 +37,60 @@ Response:
 {}`
 
 type EditUserCmd struct {
-	EmailNotifications *uint64 `long:"emailnotifications" optional:"true" description:"Whether to notify via emails"`
+	Args struct {
+		NotifType string `long:"emailnotifications" description:"Email notifications"`
+	}
 }
 
 func (cmd *EditUserCmd) Execute(args []string) error {
+
+	EmailNotifs := map[string]v1.EmailNotificationT{
+		"userproposalchange":        1,
+		"userproposalvotingstarted": 2,
+		"proposalvetted":            4,
+		"proposaledited":            8,
+		"votingstarted":             16,
+		"newproposal":               32,
+		"userauthorizedvote":        64,
+		"commentonproposal":         128,
+		"commentoncomment":          256,
+	}
+
+	var notif v1.EmailNotificationT
+	a, err := strconv.ParseUint(cmd.Args.NotifType, 10, 64)
+	if err == nil {
+		// Numeric action code found
+		notif = v1.EmailNotificationT(a)
+	} else if a, ok := EmailNotifs[cmd.Args.NotifType]; ok {
+		// Human readable action code found
+		notif = a
+
+	} else if a, ok := EmailNotifs[strings.Split(cmd.Args.NotifType, ",")[0]]; ok {
+		// List of human readable action codes found
+
+		notif = a
+		// Parse list of strings and calculate associated integer
+		s := strings.Split(cmd.Args.NotifType, ",")
+		for i := 1; i < len(s); i++ {
```

Comment:
Would prefer range.

```
for _, v := range s {

}
```

Reply:
Testing this I actually found a bug:/  In my old for loop, I was starting the indexing of the array at '1' instead of '0'. This is probably from writing a bunch of Matlab code back in the day (which indexes arrays starting at '1' instead of '0'). Used the suggested for loop and am getting the correct numbers now.